### PR TITLE
Misc Bullseye fixes

### PR DESCRIPTION
**Pull request by singingtelegram**

### CODE REVIEW

File: modules/ocf_desktop/manifests/packages.pp [hunk 1]
```diff
@@ -93,6 +93,9 @@
 
         # Matchbox is what we use on our RPi
         'matchbox-keyboard',
+
+        # x4vncviewer is no longer present
+        'tigervnc-viewer',
```
Comment (ethanhs) [inline modules/ocf_desktop/manifests/packages.pp:98]: Does ocf-tv use x4vncviewer or does it use tigervnc-viewer?

File: modules/ocf_desktop/manifests/xsession.pp [hunk 2]
```diff
@@ -129,14 +129,26 @@
...
-      target  => '/opt/share/xsession/images/ocf-color-256.png',
-      require => File['/opt/share/xsession/images'];
-  }
+  if $::lsbdistcodename == 'bullseye' {
+    file {
+      ['/usr/share/icons/Adwaita', '/usr/share/icons/Adwaita/512x512', '/usr/share/icons/Adwaita/512x512/status']:
+        ensure => directory;
+      '/usr/share/icons/Adwaita/512x512/status/avatar-default.png':
+        ensure  => link,
+        target  => '/opt/share/xsession/images/ocf-color-512.png',
```
Comment (ethanhs) [inline modules/ocf_desktop/manifests/xsession.pp:138]: This is an image we provide AIUI. So I think this should stay 256 and not change to 512?

**Review by ethanhs — COMMENTED** [review]
These changes seem pretty reasonable. Just a couple of questions about the xsession theme and vnc/ocf-tv

Thanks!

**Comment by singingtelegram** [comment]
xvnc4viewer (sorry for the typo) is a transitional package available on
stretch and buster only, and it points to tigervnc-viewer.

Re: the 512px logo, it seems that lightdm-greeter uses the 512x512 avatar
on bullseye, hence the if clause.

**Comment by ethanhs** [comment]
> xvnc4viewer (sorry for the typo) is a transitional package available on stretch and buster only, and it points to tigervnc-viewer.

gotcha, ok that makes sense

>Re: the 512px logo, it seems that lightdm-greeter uses the 512x512 avatar on bullseye, hence the if clause.

Sorry I probably wasn't clear, but the name of the file we add is still `ocf-color-256.png`, right? So it won't find the image if we say to look for `ocf-color-512.png`.

**Comment by singingtelegram** [comment]
I added a 512px one, should be in the files changed tab

**Comment by ethanhs** [comment]
> I added a 512px one, should be in the files changed tab

Oh yeah I totally missed that. Would be nice if github gave a preview of some kind...

**Review by ethanhs — APPROVED** [review]

**Pull request merged**